crate/pages/install/local.html

```diff
@@ -1,4 +1,8 @@
 title: Install on your local machine
+description: Download Crate. Easy installation of Crate in the cloud, docker, *nix and OSX. Works with Java, Python, Ruby, PHP, Node.js, erlang, Mono, Golang, and dbext
+stable_version: 0.49.6
```

Comment:
Yes @chaudum or the link and the version text don't show properly, currently they don't show on the production site. The description isn't, but there's no real harm in leaving it there.